### PR TITLE
weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
       "amd64OnlyVersions": [
         "v2.0.0",
+        "v2.0.0",
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:14]: why dup entries.

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -122,15 +125,18 @@
       ],
       "multiArchVersions": [
         "v1.0.0.2",
-        "v1.0.0.3"
+        "v1.0.0.3",
+        "v1.1.1.3",
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:129]: curious why v1.1.1.3 and v1.1.1.4 are both in use.

**Review by andyliuliming — CHANGES_REQUESTED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json]: there seem to be some issue in dedup process when there is a leading `v` in image tag
fixed this PR with a new commit for now

File: vhdbuilder/packer/components.json (hunk 2 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json]: I think v1.1.1.3 is in use but not preloaded,
while v1.1.1.4 is latest patch version

**Review by andyliuliming — APPROVED** [review]

**Pull request merged**